### PR TITLE
Smoke tests messaging on cluster

### PR DESCRIPTION
**Pull request by kornys**
### Type of change

- Enhancement

### Description

Run smoke tests only with messaging on cluster without routes/ingress
based on PR #4727

### Checklist

- [ ] Update/write design documentation in `./documentation/design`
- [ ] Write tests and make sure they pass
- [ ] Update documentation
- [ ] Check RBAC rights for Kubernetes / OpenShift roles
- [x] Try your changes from Pod inside your Kubernetes and OpenShift cluster, not just locally
- [ ] Reference relevant issue(s) and close them after merging
- [ ] Update CHANGELOG.md

### CODE REVIEW

**Comment by kornys** [comment]
@lulf because I played with kubernetes and I saw that we depend with old enmasse on routes and minikube, so I wanted to do smoke tests more kube friendly. but if you think it is useless we can close it :(

**Comment by kornys** [comment]
@enmasse-ci run tests test_only

**Comment by kornys** [comment]
@lulf for sure I triggered PR profile where we use messaging on cluster as well

**Comment by kornys** [comment]
@enmasse-ci run tests profile=smoke test_only crc

File: systemtests/src/main/java/io/enmasse/systemtest/utils/AddressSpaceUtils.java [hunk 1]
```diff
@@ -237,6 +237,24 @@ public static Endpoint getEndpointByName(AddressSpace addressSpace, String endpo
...
     }
 
+    public static Endpoint getInternalEndpointByName(AddressSpace addressSpace, String endpoint, String protocol) {
+        for (EndpointSpec addrSpaceEndpoint : addressSpace.getSpec().getEndpoints()) {
+            if (addrSpaceEndpoint.getName().equals(endpoint)) {
+                EndpointStatus status = getEndpointByName(addrSpaceEndpoint.getName(), addressSpace.getStatus().getEndpointStatuses());
+                log.debug("Got endpoint: name: {}, service-name: {}, host: {}, port: {}",
+                        addrSpaceEndpoint.getName(), addrSpaceEndpoint.getService(), status.getServiceHost(),
+                        status.getServicePorts().get(protocol));
+                if (status.getServiceHost() == null) {
```
Comment (lulf) [inline systemtests/src/main/java/io/enmasse/systemtest/utils/AddressSpaceUtils.java:247]: Should check the port as well.

Comment (kornys) [reply systemtests/src/main/java/io/enmasse/systemtest/utils/AddressSpaceUtils.java]: makes sense, done

**Pull request merged**